### PR TITLE
fix(i18n): new updates from crowdin

### PR DESCRIPTION
**Pull request by berty-assistant**

### CODE REVIEW

File: js/packages/berty-i18n/locale/zh/messages.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "time": {
-    "fuzzy-time": "",
-    "fuzzy-month-day": "",
-    "fuzzy-year-month": "",
-    "datetime": "",
+    "fuzzy-time": "时:分",
+    "fuzzy-month-day": "月/日",
+    "fuzzy-year-month": "年/月",
+    "datetime": "年-月-时:分",
```
Comment (Jorropo) [inline js/packages/berty-i18n/locale/zh/messages.json:6]: Doesn't this breaks our time lib ?

Comment (Monsterxi) [reply js/packages/berty-i18n/locale/zh/messages.json:6]: sorry? time lib?

Comment (Monsterxi) [reply js/packages/berty-i18n/locale/zh/messages.json:6]: Do you set the display time to 12 hours or 24 hours?   In China, most of them are based on 24 hours。If I understand wrong, please understand。

Comment (Jorropo) [reply js/packages/berty-i18n/locale/zh/messages.json:6]: > Do you set the display time to 12 hours or 24 hours? In China, most of them are based on 24 hours。If I understand wrong, please understand。thanks

I mean those patterns are used to setup how time is drawn (you are never gonna see `hh/MM` it's gonna be replaced by the time library).

**Review by moul — APPROVED** [review]

**Pull request merged**